environment

```diff
@@ -107,6 +107,12 @@ azul_vars=(
     # content requested by client.
     AZUL_S3_BUCKET
 
+    # URL pointing to the static website hosted at the S3 bucket for URL redirection
```

Comment:
Domain names aren't URLs. Those are distinct concepts.

Reply:
Also, isn't there a dependency between the two? They must be the same, correct? Why do we need two variables in that case.